src/Compositor.cpp

```diff
@@ -460,6 +460,8 @@ void CCompositor::cleanup() {
     m_bIsShuttingDown   = true;
     Debug::shuttingDown = true;
 
+    g_pConfigManager->dispatchExecShutdown();
```

Comment:
should be executed just before signal() a few lines up

Reply:
Execs were started too late anyways, resolved.